packages/components/src/presentation-components/instance-filter-builder/MultiTagSelect.tsx

```diff
@@ -89,7 +89,7 @@ function TagSelectOption<TOption, IsMulti extends boolean = boolean>({ children:
 
   return (
     <components.Option {...props} className={className}>
-      <span>{optionLabel === "" ? <i>Empty Value</i> : optionLabel}</span>
+      {optionLabel === "" ? <span className="presentation-instance-filter-empty-property-text">Empty Value</span> : <span>optionLabel</span>}
```

Comment:
- [ ] `Empty Value` needs to be localized.
- [ ] I'd change the class name to `presentation-instance-filter-special-property-value`.
- [ ] Could we make this class part of `className`? E.g.
   ```tsx
   const className = classnames("iui-menu-item", {
     "iui-focused": props.isFocused,
     "iui-active": props.isSelected,
     "presentation-instance-filter-special-property-value": (optionLabel === ""),
   });
   ```